grails-app/assets/javascripts/consentGroupReview.js

```diff
@@ -91513,21 +91528,21 @@ var ConsentGroupReview = function (_Component) {
   }, {
     key: 'isCurrentUserAdmin',
     value: function isCurrentUserAdmin() {
-      var _this3 = this;
+      var _this4 = this;
```

Comment:
I'm curious about why we need to rename these. Are we using these in local storage?